src/material/badge/badge-module.ts

```diff
@@ -9,10 +9,10 @@
 import {NgModule} from '@angular/core';
 import {MatCommonModule} from '@angular/material/core';
 import {A11yModule} from '@angular/cdk/a11y';
-import {MatBadge} from './badge';
+import {MatBadge, _MatBadgeStyleLoader} from './badge';
 
 @NgModule({
-  imports: [A11yModule, MatCommonModule, MatBadge],
+  imports: [A11yModule, MatCommonModule, MatBadge, _MatBadgeStyleLoader],
```

Comment:
This shouldn't be necessary since the component is standalone, but it was causing some issues in our own tests.

Reply:
Could we have a comment that explains why?

Reply:
Done.